manifests/cf-manifest/scripts/generate-cf-certs.sh

```diff
@@ -18,6 +18,9 @@ router_ssl,${ROUTER_DOMAINS}
 uaa_jwt_signing,
 consul_server,server.dc1.cf.internal,server.dc2.cf.internal
 consul_agent,
+logsearch,logsearch.${SYSTEM_DNS_ZONE_NAME}
+metrics,metrics.${SYSTEM_DNS_ZONE_NAME}
+router,${ROUTER_DOMAINS}
```

Comment:
I don't think that should be a problem.  Would we need to do anything other than update the keys the manifest is looking for here: https://github.com/alphagov/paas-cf/blob/master/manifests/cf-manifest/deployments/030-cf-jobs.yml#L879-L880